Quadriga/src/main/java/edu/asu/spring/quadriga/rest/open/NodeInfoController.java

```diff
@@ -111,4 +114,11 @@ public int compare(TextPhrase o1, TextPhrase o2) {
         return new ResponseEntity<String>(result.toString(), HttpStatus.OK);
 
     }
+
+    public void appendCall(JSONObject occurance, TextOccurance occur) throws JSONException {
```

Comment:
As this method appends JSONObject key with its value I named it that way. Should this be changed?